android/build.gradle.kts

```
@@ -66,6 +67,29 @@ android {
     }
 }
 
+koverReport {
+    val excludedFiles = listOf(
+        "io.mockative.*",
+        "*.BuildConfig",
+        "*.BuildKonfig",                        // BuildKonfig generated
+        "*.ComposableSingletons*",              // Jetpack Compose generated
+        "*.*\$*Preview\$*",                     // Jetpack Compose Preview functions
+        "*.di.*",                               // Koin
+        "*.ui.preview.*",                       // Jetpack Compose Preview providers
+        "*.*Test",                              // Test files
+        "*.*Test*",                             // Test cases
+        "*.*Mock",                              // mockative @Mock generated
+        "*.test.*",                             // Test util package
+        "*.*\$\$serializer",                    // Kotlinx serializer
+    )
+
+    filters {
+        excludes {
+            classes(excludedFiles)
+        }
+    }
+}
```

this setting should be updated to merge with `stagingDebug`

```suggestion
koverReport {
    defaults {
        mergeWith("stagingDebug")
        filters {
            val excludedFiles = listOf(
	        "io.mockative.*",
	        "*.BuildConfig",
	        "*.BuildKonfig",                        // BuildKonfig generated
	        "*.ComposableSingletons*",              // Jetpack Compose generated
	        "*.*\$*Preview\$*",                     // Jetpack Compose Preview functions
	        "*.di.*",                               // Koin
	        "*.ui.preview.*",                       // Jetpack Compose Preview providers
	        "*.*Test",                              // Test files
	        "*.*Test*",                             // Test cases
	        "*.*Mock",                              // mockative @Mock generated
	        "*.test.*",                             // Test util package
	        "*.*\$\$serializer",                    // Kotlinx serializer
            )

            excludes {
                classes(excludedFiles)
            }
        }
    }
}
```

---

@luongvo Then we need to rebase this on #9 to get the app flavor